classes/Blocks.php

```diff
@@ -38,15 +38,54 @@ function register_blocks()
         );
     }
 
+    function enqueue_cardlink_events() {
+        wp_enqueue_script('cardlink-events', plugin_dir_url(__FILE__) . '../scripts/cardlink-events.js');
```

Comment:
Might be slightly better to use `plugin_dir_url(__DIR__)` here so we already start from the parent directory, but either way it will stop working if the class file moves deeper/higher.

Reply:
Done, for all it matters.